client/src/components/Ticketing/donation/OnlyDonatePage.tsx

```diff
@@ -22,21 +27,33 @@ import {loadStripe} from '@stripe/stripe-js';
 export default function OnlyDonationPage(): ReactElement {
   const donation = useAppSelector(selectDonation);
   const [amount, setAmount] = useState(donation);
+  const [anonymous, setAnonymous] = useState(false);
+  // Determines the donation period that the user chooses
+  // 'onetime' | 'monthly' | 'quarterly';
+  const [donationPeriod, setDonationPeriod] = useState('onetime');
   const history = useNavigate();
 
+  // Numbers used for donation buttons' amounts and labels
+  const oneTimeAmounts = [25, 50, 100, 150, 250, 500];
+  // const monthlyAmounts = [5, 10, 25, 50, 100, 250];
+  // const quarterlyAmounts = [50, 100, 150, 200, 250, 500];
+
   // Replace this with your stripe public key
   const stripePromise = loadStripe(process.env.REACT_APP_PUBLIC_STRIPE_KEY);
 
   const doCheckout = async (formData: CheckoutFormInfo) => {
     const stripe = await stripePromise;
     if (!stripe) return;
-    const response = await fetch(process.env.REACT_APP_API_2_URL + '/events/checkout', {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify({cartItems: [], formData, donation: amount}),
-    });
+    const response = await fetch(
```

Comment:
Might be worth adding a pop up to communicate if checkout failed to the user, or maybe adding it as an issue to implement in the future.

Reply:
Created a ticket https://github.com/WonderTix/WonderTix/issues/539